web_hide_user_menu_item/README.rst

```diff
@@ -0,0 +1,52 @@
+=======================
+Web Hide User Menu Item
+=======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Frmm--custom-lightgray.png?logo=github
+    :target: https://github.com/qrtl/rmm-custom/tree/15.0/web_hide_user_menu_item
+    :alt: qrtl/rmm-custom
+
+|badge1| |badge2| |badge3| 
+
+This module hides some of the user menu items on the top-right corner of the webclient.
```

Comment:
Which ?

Reply:
I added the detail description in readme file. This module documentation, support, shortcuts and account menu items on the top-right corner of the webclient
<img width="500" alt="Screen Shot 2022-08-16 at 10 22 08" src="https://user-images.githubusercontent.com/45355704/184794362-f2ea7d59-3cd5-43d8-8d83-ff29e12c3d9f.png">
.